aws-wafv2-loggingconfiguration/src/main/java/software/amazon/wafv2/loggingconfiguration/ClientBuilder.java

```diff
@@ -0,0 +1,12 @@
+package software.amazon.wafv2.loggingconfiguration;
+
+import software.amazon.awssdk.services.wafv2.Wafv2Client;
+import software.amazon.cloudformation.LambdaWrapper;
+
+public class ClientBuilder {
+    public static Wafv2Client getClient() {
+        return Wafv2Client.builder()
+                  .httpClient(LambdaWrapper.HTTP_CLIENT)
+                  .build();
+      }
+}
```

Comment:
Can we use the method from `CustomerAPIClientBuilder.getClient()`? There are some retry and backoff strategies we use (as opposed to the default one provided by the SDK).

Reply:
done